custom/ewsghana/handlers/reminder.py

```diff
@@ -0,0 +1,25 @@
+from custom.ewsghana.handlers import DEACTIVATE_REMINDERS, REACTIVATE_REMINDERS, START_MESSAGE, STOP_MESSAGE
+from custom.ewsghana.handlers.keyword import KeywordHandler
+from custom.ewsghana.utils import user_needs_reminders
+
+
+class ReminderOnOffHandler(KeywordHandler):
+
+    def help(self):
+        if user_needs_reminders(self.user):
+            self.respond(DEACTIVATE_REMINDERS)
+        else:
+            self.respond(REACTIVATE_REMINDERS)
+
+    def handle(self):
+        action = self.args[0].lower()
```

Comment:
@orangejenny yep, the way it works behind the scenes is that for each of these `KeywordHandler`s, if no args are present, it automatically invokes `.help()` instead of `.handle()`